### PR TITLE
[master] Zil 4729 add eth call test

### PR DESCRIPTION
**Pull request by moboware**
## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

**Comment by moboware** [comment]
All integration test pass now 👍

File: tests/EvmAcceptanceTests/test/rpc/eth_call.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const zilliqa_helper = require("../../helper/ZilliqaHelper");
+const { hardhat } = require("hardhat");
+assert = require('chai').assert;
+
+const METHOD = 'eth_call';
+
+describe("Calling " + METHOD, function () {
+  it("should return the from eth call", async function () {
+
+    const contractFactory = await hre.ethers.getContractFactory("SimpleContract");
```
Comment (its-saeed) [inline tests/EvmAcceptanceTests/test/rpc/eth_call.js:11]: are you using this variable? :thinking:

Comment (its-saeed) [reply tests/EvmAcceptanceTests/test/rpc/eth_call.js:11]: btw, you can use `contractFactory.bytecode` as well I guess.

**Review by its-saeed — APPROVED** [review]
just one simple note

**Pull request merged**